src/app/components/filebrowseruss/filebrowseruss.component.ts

```diff
@@ -471,6 +471,10 @@ export class FileBrowserUSSComponent implements OnInit, OnDestroy {//IFileBrowse
       },
       error => {
         this.isLoading = false;
+        if (error.status == '403') { //Permission denied
+          this.snackBar.open('Failed to open: Permission denied.', 
+          'Dismiss', { duration: 5000, panelClass: 'center' });
```

Comment:
can we make this duration:5000
duration:ERROR_DURATION 
constant in some common file and import from there?

Reply:
MessageDuartion.small, medium, long as in editor could be other good option

Reply:
Yes, not a bad thing to add